qcengine/programs/orca.py

```diff
@@ -0,0 +1,298 @@
+"""
+Calls the Orca executable.
+"""
+
+import string
+import cclib
```

Comment:
`cclib` import needs to be internal to the functions since this will not be a global dependancy.